packages/core-data/package.json

```diff
@@ -30,6 +30,7 @@
 	],
 	"dependencies": {
 		"@babel/runtime": "^7.16.0",
+		"@testing-library/react": "^11.2.2",
```

Comment:
It’s a development dependency that is not necessary for the package to work. It should be listed only in the project’s `package.json` file in `devDependencies`.